packages/cli/src/commands/account/register-metadata.ts

```diff
@@ -17,21 +20,34 @@ export default class RegisterMetadata extends BaseCommand {
       required: true,
       description: 'The url to the metadata you want to register',
     }),
+    force: flags.boolean({ description: 'Ignore checks' }),
```

Comment:
Adding a bit more description on _which_ checks. (The `isAccount` check will still run)
```suggestion
    force: flags.boolean({ description: 'Ignore metadata validity checks' }),
```